DFC.Api.JobProfiles.ProfileServices/ProfileDataService.cs

```diff
@@ -24,6 +24,7 @@ namespace DFC.Api.JobProfiles.ProfileServices
 {
     public class ProfileDataService : IProfileDataService
     {
+        private const double expiry = 24;
```

Comment:
What does 24 mean? Seconds, minutes, hours, days? Something unrelated to time?

Reply:
It's 24 hours

Reply:
Could this be made more explicit? For example by renaming the variable to `expiryTimeInHours`.